.github/workflows/ci.yml

```diff
@@ -17,21 +17,21 @@ jobs:
         node-version: [20.x, 22.x, 24.x]
 
     steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
         with:
           fetch-depth: 0
 
       - name: Install pnpm
-        uses: pnpm/action-setup@v4
```

Comment:
Looks like changing the repository settings to enforce pinned actions worked :see_no_evil: 

```
Error: The action pnpm/action-setup@v4 is not allowed in mnahkies/openapi-code-generator because all actions must be pinned to a full-length commit SHA.
```